### DISCUSSION TITLE
Superhost?  Why did it never happen?

### DISCUSSION THREAD

**Discussion by hucker75** [BOINC/boinc#5414] [General]
This would be great for people like me with 10 PCs, or for people running it at their work with 500 computers.

[https://boinc.berkeley.edu/trac/wiki/SuperHost](https://boinc.berkeley.edu/trac/wiki/SuperHost)

**Comment by davidpanderson** [comment]
Computers are all different, and those differences determine what jobs and app versions they can run.
So a superhost would have to convey individual requests and replies,
i.e. it would be an HTTP proxy.  And those already exist.

**Reply by hucker75** [reply]
Can you please suggest one?  I've tried Squid, which is a badly ported program from Linux (I'm running Windows).  It worked for 2 months then gave up, no amount of reinstalling would fix it.  Tried it on another machine, it just refused to work.
